requires.txt

```diff
@@ -0,0 +1,81 @@
+botocore<1.5.71
+boto3<1.4.6,>=1.4.1
+celery<3.1.19,>=3.1.8
+click<7.0,>=5.0
+cssutils<0.10.0,>=0.9.9
+Django<1.7,>=1.6.0
+django-crispy-forms<1.5.0,>=1.4.0
+django-jsonfield<0.9.14,>=0.9.13
+django-picklefield<0.4.0,>=0.3.0
+django-sudo<3.0.0,>=2.1.0
+django-templatetag-sugar>=0.1.0
+djangorestframework<2.5.0,>=2.4.8
+email-reply-parser<0.3.0,>=0.2.0
+enum34<1.2.0,>=0.9.18
+exam>=0.5.1
+hiredis<0.2.0,>=0.1.0
+honcho<0.8.0,>=0.7.0
+kombu==3.0.35
+ipaddress<1.1.0,>=1.0.16
+loremipsum<1.1.0,>=1.0.5
+lxml>=3.4.1
+mock<1.1,>=0.8.0
+mmh3<2.4,>=2.3.1
+oauth2>=1.5.167
+percy>=0.4.5
+petname<2.1,>=2.0
+Pillow<=4.2.1,>=3.2.0
+progressbar2<3.11,>=3.10
+psycopg2
+pytest<3.2.0,>=3.1.2
+pytest-django<2.10.0,>=2.9.1
+pytest-html<1.10.0,>=1.9.0
+python-dateutil<3.0.0,>=2.0.0
+python-memcached<2.0.0,>=1.53
+python-openid>=2.2
+PyYAML<3.12,>=3.11
+raven<6.0.0,>=5.29.0
+redis<2.10.6,>=2.10.3
+requests[security]<2.19.0,>=2.18.4
+selenium==3.4.3
+simplejson<3.9.0,>=3.2.0
+six<1.11.0,>=1.10.0
+setproctitle<1.2.0,>=1.1.7
+statsd<3.2.0,>=3.1.0
+structlog==16.1.0
+sqlparse<0.2.0,>=0.1.16
+symbolic
```

Comment:
… and here, I relaxed the dependency constraint to allow the versions we force-installed.

This should fix this error from sentry boot:

```
un 13 14:34:57 infra-errors001 sentry[8979]: Traceback (most recent call last):
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/bin/sentry", line 6, in <module>
Jun 13 14:34:57 infra-errors001 sentry[8979]:     from pkg_resources import load_entry_point
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/share/python/sentry/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2927, in <module>
Jun 13 14:34:57 infra-errors001 sentry[8979]:     @_call_aside
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/share/python/sentry/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2913, in _call_aside
Jun 13 14:34:57 infra-errors001 sentry[8979]:     f(*args, **kwargs)
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/share/python/sentry/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2940, in _initialize_master_working_set
Jun 13 14:34:57 infra-errors001 sentry[8979]:     working_set = WorkingSet._build_master()
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/share/python/sentry/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 637, in _build_master
Jun 13 14:34:57 infra-errors001 sentry[8979]:     return cls._build_from_requirements(__requires__)
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/share/python/sentry/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 650, in _build_from_requirements
Jun 13 14:34:57 infra-errors001 sentry[8979]:     dists = ws.resolve(reqs, Environment())
Jun 13 14:34:57 infra-errors001 sentry[8979]:   File "/usr/share/python/sentry/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 829, in resolve
Jun 13 14:34:57 infra-errors001 sentry[8979]:     raise DistributionNotFound(req, requirers)
Jun 13 14:34:57 infra-errors001 sentry[8979]: pkg_resources.DistributionNotFound: The 'symbolic<2.0.0,>=1.1.0' distribution was not found and is required by sentry